pkg/resource/logging-credentials/reconciler.go

```diff
@@ -0,0 +1,76 @@
+package loggingcredentials
+
+import (
+	"context"
+	"fmt"
+
+	loggedcluster "github.com/giantswarm/logging-operator/pkg/logged-cluster"
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	apimachineryerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+// Reconciler implements a reconciler.Interface to handle
+// Logging Credentials: store and maintain logging credentials
+type Reconciler struct {
+	client.Client
+	Scheme *runtime.Scheme
+}
+
+// ReconcileCreate ensures a secret exists for the given cluster.
+func (r *Reconciler) ReconcileCreate(ctx context.Context, lc loggedcluster.Interface) (ctrl.Result, error) {
+	logger := log.FromContext(ctx)
+
+	logger.Info(fmt.Sprintf("loggingcredentials checking secret %s/%s", LoggingCredentialsSecretMeta(lc).Namespace, LoggingCredentialsSecretMeta(lc).Name))
```

Comment:
We normally log using values instead of formatting strings. [Check this out](https://github.com/go-logr/logr#but-i-really-want-to-use-a-format-string).

Reply:
Fixed in commit a4be7f073961511a9b3e9630e2d222817f2faf5c